packages/twenty-server/src/engine/metadata-modules/object-metadata/object-metadata.resolver.ts

```diff
@@ -86,6 +86,8 @@ export class ObjectMetadataResolver {
     @Parent() objectMetadata: ObjectMetadataDTO,
     @Context() context: I18nContext,
   ): Promise<string> {
+    // why does icon need to be overridden?
```

Comment:
Overriding isn't linked to locale. 
In v2 we should have `translations` as a field on objectMetadata, fieldMetadata, views, etc. offering the ability for people to manage translations also for custom object (e.g. company with users in many countries).

The overriding functionality solves an other problem: if some fields of a standard object can still be edited by a user, then how do we make sure it doesn't get erased, or that we can come back to the standard value later on if we want?